apps/main/src/layout/default/Header.tsx

```diff
@@ -57,7 +58,7 @@ export const Header = ({ sections }: HeaderProps) => {
         () =>
           _parseRouteAndIsActive(
             [
-              ...(hasRouter && (!network || network?.showRouterSwap)
+              ...(hasRouter && (!network || network.showRouterSwap)
```

Comment:
Still puzzles me how the linter doesn't catch these needless null checks.

Reply:
Well this was not bad just redundant